config/internal/webhook/role_webhook.yaml.tmpl

```
@@ -0,0 +1,20 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{.WebhookName}}
+  labels:
+    app: {{.WebhookName}}
+    component: data-science-pipelines
+rules:
+  - verbs:
+      - create
+      - get
+      - list
+      - watch
+      - update
+      - patch
+      - delete
```

I think the webhook just needs read access.
```suggestion
      - get
      - list
      - watch
```